tests/e2e/e2e_upgrade_test.go

```diff
@@ -44,13 +44,9 @@ var _ = OSMDescribe("Upgrade from latest",
 			const releaseName = "osm"
 			i := action.NewInstall(helmCfg)
 
-			// Latest version excluding pre-releases used by default. Using the
-			// latest assumes we aren't maintaining multiple release branches
-			// at once. e.g. if a patch is cut for both v0.5.0 and v0.6.0, we
-			// wouldn't want to test "upgrading" backwards from v0.6.0 to
-			// v0.5.1.
+			// Install latest patch release
 			i.ChartPathOptions.RepoURL = "https://openservicemesh.github.io/osm"
-			i.Version = ">0.0.0-0" // Include pre-releases
+			i.Version = "0.9.2"
```

Comment:
Could we instead make this something like `~0.9` so then the latest 0.9.x release is picked and we don't have to touch this for future v0.9 releases? https://pkg.go.dev/github.com/Masterminds/semver/v3#hdr-Checking_Version_Constraints_and_Comparing_Versions